code/dynamic_programming/src/longest_common_subsequence/longest_common_subsequence.php

```diff
@@ -0,0 +1,30 @@
+<?php 
+
+    function lcs($x , $y) 
+    { 
+        $x_len = strlen($x); 
+        $y_len = strlen($y) ; 
+
+        for ($i = 0; $i <= $x_len; $i++) 
+        { 
+            for ($j = 0; $j <= $y_len; $j++) 
+            { 
+                if ($i == 0 || $j == 0) 
+                    $dp[$i][$j] = 0; 
+                else if ($x[$i - 1] == $y[$j - 1]) 
+                    $dp[$i][$j] = $dp[$i - 1][$j - 1] + 1; 
+                else
+                    $dp[$i][$j] = max($dp[$i - 1][$j], $dp[$i][$j - 1]); 
+            } 
+        } 
+
+        return $dp[$x_len][$y_len]; 
+    } 
+
+    $a = "AGGTAB"; 
+    $b = "GXTXAYB";
+
+    echo "Length of LCS : "; 
+    echo lcs($a, $b); 
+
+?> 
```

Comment:
Don't have this.

Reply:
Done

Reply:
@rbjoshi1309 Still need to remove line 28. Also unindent the rest of the file.

Reply:
Done

Reply:
@rbjoshi1309 You still haven't removed the trailing `?>`.

Reply:
@arnavb Done.